site/docs/pages/wallet/wallet.mdx

```diff
@@ -86,6 +87,7 @@ export function WalletComponents() {
       <WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com">
         Wallet
       </WalletDropdownLink>
+      <WalletDropdownBaseName /> 
```

Comment:
I feel like we should document this in his own page.

Reply:
@Zizzamia I'm not sure if this should have its own page. It's a very simple one line component that does not take in any new custom params: 

<img width="738" alt="Screenshot 2024-08-12 at 3 11 20 PM" src="https://github.com/user-attachments/assets/4428b56e-1b06-4c03-950c-dd3b1f18815a">